src/test/java/com/deadlinks/MainTest.java

```diff
@@ -0,0 +1,47 @@
+package com.deadlinks;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
+
+import static org.junit.Assert.*;
+
+public class MainTest {
+
+    private final ByteArrayOutputStream outputContent = new ByteArrayOutputStream();
+
+    @Before
+    public void setUp() throws Exception {
+        System.setOut(new PrintStream(outputContent));
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        System.setOut(System.out);
+    }
+
+    @Test
+    public void dead_links_test() throws JsonProcessingException {
+        String[] url ={"https://junit.org/junit5/"};
+        Main.main(url);
+        assertEquals(getExpectedValue(), outputContent.toString());
+    }
+
+    private String getExpectedValue() {
+        return "{" + System.lineSeparator() +
```

Comment:
@Velory please follow the _exact_ format described in the `README.md`.
Like there are no `urlDetails`, `totalLinks` and `deadLinks` attributes

Reply:
Could you please also move expected json to `java/test/resources/expected.json` and read it from the test instead of composing it from the code?